ostree-fdo-container.sh

```diff
@@ -0,0 +1,881 @@
+#!/bin/bash
+set -euox pipefail
+
+# Provision the software under test.
+./setup.sh
+
+# Install python3-pip
+sudo dnf install -y python3-pip
+# Install yq to modify service api server config yaml file
+sudo pip3 install yq
+
+# Set up variables.
+TEST_UUID=$(uuidgen)
+IMAGE_KEY="ostree-installer-${TEST_UUID}"
+HTTP_GUEST_ADDRESS=192.168.100.50
+PUB_KEY_GUEST_ADDRESS=192.168.100.51
+ROOT_CERT_GUEST_ADDRESS=192.168.100.52
+PROD_REPO_URL=http://192.168.100.1/repo
+PROD_REPO=/var/www/html/repo
+STAGE_REPO_ADDRESS=192.168.200.1
+STAGE_REPO_URL="http://${STAGE_REPO_ADDRESS}:8080/repo/"
+FDO_SERVER_ADDRESS=192.168.100.1
+CONTAINER_TYPE=edge-container
+CONTAINER_FILENAME=container.tar
+INSTALLER_TYPE=edge-simplified-installer
+INSTALLER_FILENAME=simplified-installer.iso
+REF_PREFIX="rhel-edge"
+# Workaround BZ#2108646
```

Comment:
This comment is not helpful in this case.

Reply:
removed.